scripts/terminalserver/terminalserver.jl

```diff
@@ -18,6 +18,26 @@ end
 include("repl.jl")
 include("../debugger/debugger.jl")
 
+function getVariables()
+    M = Main
+    variables = []
+    for n in names(M)
+        !isdefined(M, n) && continue
+        x = getfield(M, n)
+        x isa Module && continue
+        x==Main.vscodedisplay && continue
+        n_as_string = string(n)
+        n_as_string=="@run" && continue
+        n_as_string=="@enter" && continue
+        startswith(n_as_string, "#") && continue
+        t = typeof(x)
+        value_as_string = Base.invokelatest(repr, x)
```

Comment:
This should probably be `show`, no? Also would be best to `stringlimit` this, as @aviatesk mentioned below.

Reply:
Or maybe just wrap 
https://github.com/julia-vscode/julia-vscode/blob/513c5cbedfa60043c02078db6dd85652b17e988c/scripts/terminalserver/terminalserver.jl#L89
into a function and reuse that.

Reply:
This is not related to this PR, but `strlimit` already handles `isvalid` character case, so we don't need outer `isvalid` filtering, right ?

Reply:
Yeah, I just mindlessly copy-pasted code there ;)

Reply:
I had used `repr` here because that is what we use for the debugger. But I guess the situation here is different: for the debugger we ideally want the ability to round-trip things (because users can modify the value of a var in the tree view that shows vars), whereas here that is not a concern. Although: we could of course in the future add an ability to change variable values in the workspace view? In any case, I've changed it to `show` for now, we can iterate in future PRs.